packages/utils/add-script.ts

```diff
@@ -0,0 +1,20 @@
+import { json } from "mrm-core";
```

Comment:
I don't think we need a dependency to do a JSON write

Reply:
Removed 👍